components/drillList.js

```diff
@@ -37,43 +37,45 @@ export default function DrillList(props) {
   });
 
   function getDrillIndexByTitle(title) {
-    return props.drillData.findIndex((item) => item.drillType === title);
+    return drillData.findIndex((item) => item.drillType === title);
   }
 
   return (
     <SectionList
-      style={{ paddingHorizontal: 20, paddingBottom: 50 }}
+      style={{ paddingHorizontal: 20, paddingBottom: 50, height: "100%" }}
```

Comment:
I fixed some issues with bottom padding inconsistencies, so I'm not sure if the paddingBottom of 50 is still needed or not. Looks fine on latest commit either way.

I used the safeArea fix we used elsewhere in the code (add `style={{ flex: 1 }} edges={["right", "top", "left"]}` to the SafeView any time it wraps a scrollview or similar component, like FlatList)

Before:
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/e0e7dce7-905b-40dc-875b-9dc5ace6b1d2)